habitat/plan.sh

```diff
@@ -0,0 +1,205 @@
+pkg_name=chef-workstation
+pkg_origin=chef
+pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+pkg_description="Chef Workstation - Opinionated tools for getting the most out of the Chef ecosystem"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_svc_user=root
+ruby_pkg="core/ruby26"
+pkg_build_deps=(
+  core/make
+  core/gcc
+  core/gcc-libs
+  core/pkg-config
+)
+pkg_deps=(
+  core/glibc
+  core/bash
+  ${ruby_pkg}
+  core/libxml2
+  core/libxslt
+  core/xz
+  core/zlib
+  core/bundler
+  core/openssl
+  core/cacerts
+  core/libffi
+  core/libarchive
+  core/coreutils
+  core/git
+)
+
+pkg_version() {
+  cat /src/VERSION
```

Comment:
I'm still unclear about why this is superior to setting it directly in the package metadata declarations:

```
$pkg_version=$(cat /src/VERSION)
```

If it's something to do with all the things that get re-evaluated during `update_pkg_version`, I'd like to be intentional about needing that behavior.

Reply:
Also, I would like to advocate for `$PLAN_CONTEXT/../VERSION` because not every studio (particularly in dev) is entered from the root of the git repo.

Reply:
Im ok with using either `PLAN_CONTEXT` or `SRC_PATH`